### PR TITLE
Adding templates for license and readme.

### PR DESCRIPTION
**Pull request by msdemlei**
This also adds a shell script startup.sh, which will do all the
copying around of files that belong to a proper ivoatex document.

This is in the context of the github migration, where we now want to have LICENSE and README as a standard; we already had document, makefile, and .gitignore templates, so I figured it's time to automate this a bit.

The idea is that after the git add submodule, users can say sh ivoatex/startup.sh and start hacking on Makefile and $DOCNAME.tex immediately.

One thing I'd put up *some* resistance against: Including images from third-party servers -- we've had that for a cc-by logo.  It's bad enough that we sell our users' data to github.  Let's not involve even more parties just for a pretty picture.

### CODE REVIEW

File: startup.sh [hunk 1]
```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+# A script to copy templates from ivoatex when starting a document.
+
+if [ $# -ne 1 ]; then
+	echo "Usage: startup.sh <document base name>"
```
Comment (molinaro-m) [inline startup.sh:5]: is <document base name> the same as the StdProc ConciseName?

**Comment by mbtaylor** [comment]
I'm no great fan of markdown but it seems to be in much wider use in and around github than ReStructured Text (which I'm not even sure I've heard of).  People aren't generally going to be doing complicated formatting either way, so I don't feel the "better defined" argument counts for much, you just need to be able to slap in the odd heading, itemised list and link.  I don't much want to have to remember/bookmark syntax for yet another simple markup language. So put me down as -1 for RST.

**Comment by Zarquan** [comment]
If we do need to have a script to initialise the files, it should be called `init.sh` rather than `startup.sh`.
* `init.sh`is something that is run only once
* `startup.sh` is something that gets run each time the user starts using it ?

**Comment by msdemlei** [comment]
On Tue, Mar 08, 2022 at 03:33:16AM -0800, Zarquan wrote:
> If we do need to have a script to initialise the files, it should be called `init.sh` rather than `startup.sh`.
> * `init.sh`is something that is run only once
> * `startup.sh` is something that gets run each time the user starts using it ?

I follow this reasoning, but init.sh doesn't strike me as a file name
very clearly indicating its function either.  What about
make-templates.sh?  I'm of course open to better suggestions.

**Comment by msdemlei** [comment]
On Tue, Mar 08, 2022 at 01:54:23AM -0800, Marco Molinaro wrote:
> @molinaro-m commented on this pull request.
> > @@ -0,0 +1,24 @@
> +if [ $# -ne 1 ]; then
> +	echo "Usage: startup.sh <document base name>"
> 
> is <document base name> the same as the StdProc ConciseName?

Conceptually, yes.  The ivoatexDoc instructions say to pass in what's
called $DOCNAME there, and that is what the document will be called
in the end, and that's what ivoatex will use in its archive names.
I'd guess, however, this is too messy to explain in a usage message
-- if someone is in doubt, I'm afraid they'll have to skim
ivoatexDoc (which I expect they will already be doing if they call
that script).

**Comment by msdemlei** [comment]
On Tue, Mar 08, 2022 at 02:03:45AM -0800, Mark Taylor wrote:
> in the odd heading, itemised list and link.  I don't much want to
> have to remember/bookmark syntax for yet another simple markup
> language. So put me down as -1 for RST.

So, does anyone want to come out in favour of it?

**Comment by Zarquan** [comment]
> I follow this reasoning, but init.sh doesn't strike me as a file name very clearly indicating its function either. What about make-templates.sh? I'm of course open to better suggestions.

I was thinking of something like `git init` which does a similar thing for a git repository, but happy with `make-templates.sh`. 

Although, now you mention it, could this functionality be incorporated into the Makefile ?

**Comment by msdemlei** [comment]
On Tue, Mar 08, 2022 at 06:25:46AM -0800, Zarquan wrote:
> I was thinking of something like `git init` which does a similar
> thing for a git repository, but happy with `make-templates.sh`. 
> 
> Although, now you mention it, could this functionality be
> incorporated into the Makefile ?

I considered that; having rules like

  .gitignore: ivoatex/.gitignore
    cp $^ $@

or so won't cut it, though, because all these files are templates and
should not be touched if they exist, regardless of whether ivoatex
has updates.  Explaining that robustly to make is more hassle than
it's worth, I think.

One *might* write the shell script as a (dependency-less) make rule,
of course.  But you couldn't easily define the shell function, and
without that, the whole thing gets dramatically much uglier.  Well,
you *could* define the shell function by essentially inlining the
script in a single logical make line, but that will be a nightmare of
escaping whitespace and dollars.

And I don't think that buys anything: if a system doesn't have
anything that cannot execute the shell script, make won't move
either, so it's not like this adds a dependency to ivoatex.

Finally, just putting

  init:
    sh bin/make-templates.sh

into ivoatex/Makefile again won't buy you anything: When people need
to type the command, there's no Makefile in their document directory
yet and "make init" would just error out.

**Comment by msdemlei** [comment]
On Tue, Mar 08, 2022 at 02:03:45AM -0800, Mark Taylor wrote:
> I'm no great fan of markdown but it seems to be in much wider use
> in and around github than ReStructured Text (which I'm not even
> sure I've heard of).  People aren't generally going to be doing

Uh!  You're missing out!

> complicated formatting either way, so I don't feel the "better
> defined" argument counts for much, you just need to be able to slap
> in the odd heading, itemised list and link.  I don't much want to
> have to remember/bookmark syntax for yet another simple markup
> language. So put me down as -1 for RST.

Well, I had promised to put up no resistance, so with commit
ae22448e, the README is now in markdown.  Suggestions for how to
improve this are still welcome.

While I'm speaking: commit 652a2d436 renames startup.sh to
make-templates.sh as per Dave's request.  I've updated the
corresponding ivoatexDoc PR already, or so I hope.

**Pull request merged**